README.md

```diff
@@ -131,7 +131,26 @@ If your implementation supports multiple versions, run the above procedure for e
 
 1. The suite, notably in its `refRemote.json` file in each draft, expects a number of remote references to be configured.
    These are JSON documents, identified by URI, which are used by the suite to test the behavior of the `$ref` keyword (and related keywords).
-   Depending on your implementation, you may configure how to "register" these either by retrieving them from the `remotes/` directory at the root of the repository, *or* you may execute `bin/jsonschema_suite remotes` using the executable in the `bin/` directory, which will output a JSON object containing all of the remotes combined.
+   Depending on your implementation, you may configure how to "register" these *either*:
+
+    * by directly retrieving them off the filesystem from the `remotes/` directory, in which case you should resolve any `$ref` in the directory relative to a base URI of `http://localhost:1234` such that a `$ref` to `http://localhost:1234/foo/bar/baz.json` resolves to the file at `remotes/foo/bar/baz.json`
```

Comment:
Would it be better to say something along the lines "in which case you should _load each schema with a retrieval URI_ of `http://localhost:1234/` followed by the relative path from the remotes directory"?  The base URI resolution flows form that, it shouldn't be possible to set it separately on a per-`$ref`-resolution basis which is what this kind of sounds like.

Reply:
yes, that wording makes more sense to me - also regarding "you should resolve any `$ref` in the directory", I assume 'the directory' to mean `remotes/` but most the $refs this applies to come from `tests/`.

for the end of the sentence, maybe "resolves to the schema loaded from the file `remotes/foo/bar/baz.json`"

Reply:
> for the end of the sentence, maybe "resolves to the schema loaded from the file `remotes/foo/bar/baz.json`"

Assuming the retrieval URI (and therefore the base URI) are et correctly, and the remotes pre-loaded, there shouldn't be anything special needed to make that happen.  Unless I'm misunderstanding something here.

Reply:
it's not a hugely significant distinction between the present wording and my suggestion - "resolving to the file" vs. "resolving to the schema loaded from the file". just that, at least in my implementation, by the time the $ref is resolved, the schema has to have been loaded and registered, and the file is irrelevant and unknown.

Reply:
Thanks! Makes sense, I made a tweak here more in line with the above, have another look.